examples/statistics/histogram_demo_cumulative.py

```diff
@@ -1,29 +1,67 @@
 """
-Demo of the histogram (hist) function used to plot a cumulative distribution.
+==========================================================
+Demo of using histograms to plot a cumulative distribution
+==========================================================
+
+This shows how to plot a cumulative, normalized histogram as a
+step function as means of visualization the empirical cumulative
+distribution function (CDF) of a sample. We also use the `mlab`
+module to show the theoretical CDF.
+
+A couple of other options to the `hist` function are demostrated.
+Namely, we use the `normed` parameter to normalize the histogram and
+a couple of different options to the `cumulative` parameter. Normalizing
+a histogram means that the counts within each bin are scaled such that
+the total height of each bin sum up to 1. Since we're showing the
+cumulative histogram, the max value at the end of the series is 1.
+The `normed` parameter takes a boolean value.
+
+The `cumulative` kwarg is a little more nuanced. Like `normed`, you can
+pass it True or False, but you can also pass it -1 and that will
+reverse the distribution. In engineering, CDFs where `cumulative` is
+simply True are sometimes "non-excedance" curves. In other words, you
```

Comment:
A quick Google shows excedance used in what appears to be a different context. I think this one is exceedance.